src/include/stat.h

```diff
@@ -686,6 +686,11 @@ struct __wt_connection_stats {
     int64_t txn_checkpoint_time_max;
     int64_t txn_checkpoint_time_min;
     int64_t txn_checkpoint_time_recent;
+    int64_t txn_checkpoint_prep_running;
+    int64_t txn_checkpoint_prep_max;
+    int64_t txn_checkpoint_prep_min;
+    int64_t txn_checkpoint_prep_recent;
+    int64_t txn_checkpoint_prep_total;
```

Comment:
You need to rerun `s_all` to allow all the auto-generated files to be re-alphabetized too.

Reply:
It seems that `s_all` isn't alphabetising correctly - running after the `stat_data.py` reordering it isn't changing any files. @sueloverso is seeing this same behaviour.